.github/workflows/scripts/aot-demo.sh

```diff
@@ -148,4 +148,23 @@ function build-and-test-headless-demo {
     compare_to_groundtruth android
 }
 
+function build-and-test-headless-demo-desktop {
+    pushd taichi
+    setup_python
+    python3 -m pip install dist/*.whl
+    sudo chmod 0777 $HOME/.cache
+    export TAICHI_REPO_DIR=$(pwd)
+    popd
+
+    rm -rf taichi-aot-demo
+    #git clone --recursive --depth=1 https://github.com/taichi-dev/taichi-aot-demo
+    git clone --recursive https://github.com/jim19930609/taichi-aot-demo.git
```

Comment:
aha I usually just push my change to a branch in taichi-dev/taichi-aot-demo so that I can just add a `-b my_branch` to swtich ;) either works :D